ml/Dimension.h

```diff
@@ -88,6 +88,8 @@ class PredictableDimension : public TrainableDimension {
 
     bool isAnomalous() { return AnomalyBit; }
 
+    double anomalousBitCount;
```

Comment:
Nit: the first letter should be capitalized for object fields and variables. Functions/Methods should start with a lower-case letter.

Reply:
ok, I must be mixing with misrac standards